### PR TITLE
tighten authorization rules

### PR DESCRIPTION
**Pull request by deads2k**
Removes cluster-admin rights from system:authenticated and system:unauthenticated.

### CODE REVIEW

**Comment by deads2k** [comment]
[test]

**Comment by deads2k** [comment]
@smarterclayton @liggitt fish or cut bait.

**Comment by deads2k** [comment]
2/3 on travis.

**Comment by smarterclayton** [comment]
So what README and documentation plans do you have changed so that once this happens every openshift user who tries it isn't broke?

**Comment by liggitt** [comment]
Should update the help shown in the web UI when no projects exist to tell them to run `new-project --admin <current user>`

**Comment by deads2k** [comment]
> Should update the help shown in the web UI when no projects exist to tell them to run new-project --admin <current user>

Ok.  I guess I'll also craft something for the mailing list and maybe add instructions to set up a user capable of viewing `default` for the sample app

**Comment by derekwaynecarr** [comment]
Doesnt the sample app use 'test'?

**Comment by deads2k** [comment]
It uses both.  `default` for the docker registry and `test` for the rest.

**Comment by deads2k** [comment]
@liggitt I updated the readme with a little more detail.  Care to make sure it works for you?

**Comment by smarterclayton** [comment]
Not going to merge this until we cut another release image (v0.3.2)

**Comment by deads2k** [comment]
Rebased.  Removed the readme updates since they were superceded by #1104 which can go in separately.

**Comment by smarterclayton** [comment]
Does everyone know this is coming and how to react?

**Comment by deads2k** [comment]
I'll send out a note now.

**Comment by deads2k** [comment]
> Does everyone know this is coming and how to react?

Yes, everyone knows this is coming.  Basic instructions were provided in an email and link to our more complete documentation was also provided.  I also included information about gathering policy and bindings for bug reports (not that I think anything could possibly go wrong :) )

**Comment by liggitt** [comment]
just noticed the Jenkins example isn't using identity yet (there's a bug open for cert validation errors, but identity would hit it also). Wondering if we should wait until we have a good way to inject a kubeconfig as a secret.

**Comment by smarterclayton** [comment]
Up to Ben

**Comment by deads2k** [comment]
@bparees see above.

**Comment by deads2k** [comment]
Bug https://bugzilla.redhat.com/show_bug.cgi?id=1196022 found.

Do not merge this until we get https://github.com/openshift/origin/pull/1140

**Comment by bparees** [comment]
Not only do the jenkins setup steps need to be fixed to use identity, the jenkins job itself would need to have the certs (because it runs osc commands too) which it currently does not, so there's significant rework needed.  (or use the --insecure flag, anyway, after doing an osc login or something).

I haven't had a lot of people asking me questions about the jenkins example so my guess is it's not getting a lot of attention, but it would be nice to keep it working.

**Comment by deads2k** [comment]
> but it would be nice to keep it working.

It's currently broken, so it would be "make it work again".

**Comment by bparees** [comment]
@deads2k is that the sound of volunteering i hear?

**Comment by smarterclayton** [comment]
It's definitely something people are looking at. Let's create a maint card if it still needs work after David's change.

**Comment by bparees** [comment]
I have a pr in progress to get it working again. That's where I hit the duplicate tag imagerepo issue. 

Ben Parees | OpenShift


-----Original Message-----

**Comment by liggitt** [comment]
reminder to myself to make sure we have the jenkins example working with secrets before merging this

**Comment by brenton** [comment]
@deads2k, @bparees, @liggitt, Is this realistic for beta2 at this point?  Some of the other things Erik is planning to demo need this.

**Comment by liggitt** [comment]
if we tighten this, the core path (master/node/registry/router) works, but other things like the Jenkins example will stop working until we can get identity info into it. We're planning to do that with secrets, but those won't be in origin until the rebase merges, and then there is some follow-up work to make creating a secret for Jenkins to use reasonable for an end-user.

tl;dr - this can go in now if we accept things like the Jenkins example breaking until we get secret support into origin.

**Comment by smarterclayton** [comment]
That's fine with me

**Comment by brenton** [comment]
I saw the menion about jenkins in the comment.  The beta training isn't making use of jenkins right now so I think we'd be OK.  @thoraxe (let us know if you have concerns)

**Comment by thoraxe** [comment]
I'm fine with Jenkins not working for beta2.

**Comment by liggitt** [comment]
pull the trigger! [merge]
